src/app/utils/index.ts

```diff
@@ -78,3 +78,16 @@ export async function getAlbyWalletOptions() {
     console.error(e);
   }
 }
+
+// to extract lightning data associated with the lightning tag within the URL. eg. LNBits QR codes
+export function extractLightningTagData(url: string) {
```

Comment:
Is this only used in one place? does it make sense to be moved back to the Send screen?

Reply:
shall we use this in redeemLNURL screen as well!. if user pastes the url directly in the form field this function will filter and get use withdraw code there as well?

Reply:
currently adding it